### PR TITLE
%s changed to fstring, single quotes in f string changed to double qu…

### PR DESCRIPTION
**Pull request by Behemkot**
https://github.com/pandas-dev/pandas/issues/29547#issuecomment-562164065
https://github.com/pandas-dev/pandas/issues/29547

- [x] pandas/_libs/tslibs/frequencies.pyx
- [x] pandas/_libs/tslibs/period.pyx
- [x] pandas/_libs/tslibs/strptime.pyx

changed %s to f strings and f ' '  to f " ".
str.format() remained unchanged due to:
https://github.com/pandas-dev/pandas/issues/29547#issuecomment-562227987

### CODE REVIEW

File: pandas/_libs/tslibs/period.pyx [hunk 1]
```diff
@@ -1209,8 +1209,7 @@ def period_format(int64_t value, int freq, object fmt=None):
         elif freq_group == 4000:  # WK
             left = period_asfreq(value, freq, 6000, 0)
             right = period_asfreq(value, freq, 6000, 1)
-            return '%s/%s' % (period_format(left, 6000),
-                              period_format(right, 6000))
+            return f"(period_format(left, 6000)/period_format(right, 6000))"
```
Comment (jbrockmendel) [inline pandas/_libs/tslibs/period.pyx]: are we calling this function inside a loop?  if so, we may need to be careful about whether cython fstrings differently from % formatting

Comment (jbrockmendel) [reply pandas/_libs/tslibs/period.pyx]: more generally, id prefer to have non-trivial function calls occur outside of f-space.

Comment (jbrockmendel) [reply pandas/_libs/tslibs/period.pyx]: the CI failures look related to this

**Comment by alimcmaster1** [comment]
https://github.com/pandas-dev/pandas/pull/30094 can you submit updates on the same PR opposed to a new PR in the future

File: pandas/_libs/tslibs/period.pyx (hunk 1 above)
Comment (WillAyd) [inline pandas/_libs/tslibs/period.pyx]:
```suggestion
            return f"{period_format(left, 6000}/{period_format(right, 6000}"
```

**Review by WillAyd — CHANGES_REQUESTED** [review]

File: pandas/_libs/tslibs/period.pyx (hunk 1 above)
Comment (jbrockmendel) [reply pandas/_libs/tslibs/period.pyx]: just pushed this change.  it fixed the errors for locally

**Comment by jbrockmendel** [comment]
LGTM

**Comment by jbrockmendel** [comment]
over to you @WillAyd

**Comment by WillAyd** [comment]
Thanks @Behemkot and @jbrockmendel

**Pull request merged**